### ISSUE TITLE
User creation/login

### ISSUE THREAD

**Issue by tmess11** [antebrl/IPTV-Restream#35]
### Problem to solve

Users have a random username when launching the application on the WebUI.

### Proposal

I suggest being able to make users and assign passwords to allow users to persist across platforms and instances as well as restrict access to the application.

**Comment by digitalSaj** [comment]
This would be very useful. Also if some rights can be added, eg. users with channel addition rights can only add channels, maybe another type for channel change.

**Comment by aniel300** [comment]
does this one goes in hand with https://github.com/antebrl/IPTV-Restream/issues/43

**Comment by antebrl** [comment]
Yes, definetely. I'll resolve these issues together in one pull request.

**Comment by aniel300** [comment]
thank u bud, no rush

**Comment by rob-fi** [comment]
Just chiming in to say I'd highly appreciate this feature. I currently use hls-proxy, but the web UI is clunky and it also doesn't allow user login - I have to whitelist friend's IP addresses. I want to share some streams with a few friends but have them all be able to watch different channels (which goes in hand with the goal for multiple stream support) but not allow random people/bots to be able to access the server.
